### DISCUSSION TITLE
AAAD Android -Car Stream Regarding

### DISCUSSION THREAD

**Discussion by vaseeharanjohn** [community/community#140953] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

AAAD Android -Car Stream Regarding - in my previous android version .ie. below android 13 is working car stream for my mobile and car head unit, now my mobile updated the Android 14, but not able to use car stream. please help me.  *removed PII* and *removed PII* from INDIA
*PII removed by @LiteBrite82*

**Comment by LiteBrite82** [comment]
Hi @vaseeharanjohn ,

I've removed your private information from [this post](link to discussion) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
